.expeditor/release.omnibus.yml

```diff
@@ -11,14 +11,8 @@ builder-to-testers-map:
     - debian-8-x86_64
     - debian-9-x86_64
     - debian-10-x86_64
-  debian-10-aarch64:
-    - debian-10-aarch64
   el-6-x86_64:
     - el-6-x86_64
-  el-7-aarch64:
```

Comment:
don't pull this out, the el7 builder and its testers work just fine

it is just debian, sles15, and ubuntu-18.04 aarch64 that need `gem "unf_ext", "~> 0.0.7.6"`

as evidenced here:

https://buildkite.com/chef/inspec-inspec-master-omnibus-adhoc/builds/136

Reply:
this build shows them all green:

https://buildkite.com/chef/inspec-inspec-master-omnibus-adhoc/builds/131

if we add this change:

https://github.com/inspec/inspec/commit/e3e68d5fe1f7948e91069587317479cb485374b9

but then the verify pipepline fails